turbine/src/cluster_nodes.rs

```diff
@@ -362,7 +362,15 @@ impl<T: 'static> ClusterNodesCache<T> {
         working_bank: &Bank,
         cluster_info: &ClusterInfo,
     ) -> Arc<ClusterNodes<T>> {
-        let epoch = root_bank.get_leader_schedule_epoch(shred_slot);
+        let epoch = if check_feature_activation(
+            &feature_set::revise_turbine_epoch_stakes::id(),
+            shred_slot,
+            root_bank,
```

Comment:
seems like it's ok in this case that the feature activation is observed after the actual epoch boundary, since schedule and activation status are queried from the same bank?

Reply:
nope. https://github.com/solana-labs/solana/blob/b44c9bca898122bd3b6de24770c2a0acd31da222/runtime/src/bank.rs#L1539

Reply:
that is not epoch-schedule though.
did you mean epoch-stakes in your first comment?

Reply:
ah right.  should be fine so long as we don't happen to be re-parameterizing the epoch schedule via feature gate at the same time